src/BitpandaEcosystemToken.sol

```diff
@@ -117,23 +119,13 @@ contract BitpandaEcosystemToken is
     }
 
     /**
-     * @dev See {ERC20Capped-_mint}
+     * @dev See {ERC20-_update}.
      */
-    function _mint(
-        address account,
-        uint256 amount
-    ) internal override(ERC20, ERC20Capped) {
-        ERC20Capped._mint(account, amount);
-    }
-
-    /**
-     * @dev See {ERC20Pausable-_beforeTokenTransfer}
-     */
-    function _beforeTokenTransfer(
+    function _update(
         address sender,
         address recipient,
         uint256 amount
-    ) internal override(ERC20, ERC20Pausable) {
-        ERC20Pausable._beforeTokenTransfer(sender, recipient, amount);
+    ) internal override(ERC20, ERC20Capped, ERC20Pausable) {
+        super._update(sender, recipient, amount);
```

Comment:
We should not use `super` when overriding a function defined in multiple base contracts since we then depend on the order of the contracts. We should rather explicitly state the contract.

Reply:
That was my understanding, but after reading following docs/links and testing the order with both approaches, I used `super`. 
Basically,  Oz forum suggests to use `super`. So, I tested to see the behavior to see if it is calling the base classes in order of "most derived" to “most base-like” using its. And it actually calls them in desired order (see test 1 below).

Links:
1. [Best practice on using multiple inheritances of ERC extensions](https://forum.openzeppelin.com/t/best-practice-on-using-multiple-inheritances-of-erc-extensions/27619)
2. https://solidity-by-example.org/inheritance/
3. [Multiple Inheritance and Linearization](https://docs.soliditylang.org/en/v0.8.26/contracts.html#multiple-inheritance-and-linearization)

Test 1 and 2 gives similar result (as we need), test 3 shows that using explicit call may skip a base class.

Test- `BitpandaEcosystemTokenTest::test_pantosTransferTo`

1. super._update(sender, recipient, amount);
```
    ├─ [34097] BitpandaEcosystemTokenHarness::pantosTransferTo(0x0000000000000000000000000000000000000002, 1000)
    │   ├─ [0] console::log("PantosBaseToken.pantosTransferTo") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("BitpandaEcosystemToken._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20Pausable._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20Capped._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ emit Transfer(from: 0x0000000000000000000000000000000000000000, to: 0x0000000000000000000000000000000000000002, value: 1000)
    │   └─ ← [Stop] 
```

4. ERC20Pausable._update(sender, recipient, amount);
```
[34097] BitpandaEcosystemTokenHarness::pantosTransferTo(0x0000000000000000000000000000000000000002, 1000)
    │   ├─ [0] console::log("PantosBaseToken.pantosTransferTo") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("BitpandaEcosystemToken._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20Pausable._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20Capped._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ emit Transfer(from: 0x0000000000000000000000000000000000000000, to: 0x0000000000000000000000000000000000000002, value: 1000)
    │   └─ ← [Stop] 
```
5. ERC20Capped._update(sender, recipient, amount); 
```
    ─ [33254] BitpandaEcosystemTokenHarness::pantosTransferTo(0x0000000000000000000000000000000000000002, 1000)
    │   ├─ [0] console::log("PantosBaseToken.pantosTransferTo") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("BitpandaEcosystemToken._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20Capped._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ [0] console::log("ERC20._update") [staticcall]
    │   │   └─ ← [Stop] 
    │   ├─ emit Transfer(from: 0x0000000000000000000000000000000000000000, to: 0x0000000000000000000000000000000000000002, value: 1000)
    │   └─ ← [Stop] 
```